src/screens/PutawayItem/index.tsx

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable no-shadow */
+/* eslint-disable react-native/no-inline-styles */
```

Comment:
If there are inline styles, please move to `styles` file

Reply:
we planned to create a common global style in the app right now you can ignore this file for inline style